.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,7 +12,7 @@
 - [ ] I signed the [CLA].
 - [ ] I listed at least one issue that this PR fixes in the description above.
 - [ ] I updated/added relevant documentation (doc comments with `///`).
-- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
+- [ ] I added new tests to check the change I am making, or this PR is test-exempt as per the [Tree Hygiene] page rules.
```

Comment:
Maybe link this directly to https://github.com/flutter/flutter/wiki/Tree-hygiene#tests since it's a very long page?

Reply:
There's a trade-off between being specific here and making the list below even longer, on an already-long PR template...

There's a checkbox earlier that confirms that they read the whole doc already, so presumably they know what it says? :-)

Reply:
> There's a checkbox earlier that confirms that they read the whole doc already, so presumably they know what it says? :-)

In which case we don't need this checbox, right? 🙂 

I'm not sure I follow what the cost of making a non-displayed final section longer is.

Reply:
It's visible when you have to edit the box -- it's already quite long. But yeah, I don't know what the trade-off should be here. I'll add the link.